app/controllers/PostsController.php

```diff
@@ -0,0 +1,121 @@
+<?php
+namespace app\controllers;
+use app\models\Comment;
+use app\models\Post;
+use app\utils\View;
+class PostsController {
```

Comment:
Would be nice to see a consistent approach to spacing prior to class/method/property names throughout the code.  It can help with readability quite a bit.  In this instance I'd like to see something like:

```
use app\utils\View;

class PostsController {
    
    private Post $post;
    private Comment $comment;
    
    public function __construct() {
        $this->post = new Post(); // Initialize the Post instance
        $this->comment = new Comment();
    }